### PR TITLE
Simplify Github issue and pull request templates

### PR DESCRIPTION
**Pull request by msulprizio**
The existing issue and pull request templates have been removed and replaced with templates that have been greatly simplified. The new templates:

1. Point users to the support and contribution guidelines, documentation, and debug pages first.
2. Include a section for users to identify themselves and their institution since this is not always obvious from their Github accounts.
3. Remove automatic tags and bracketed titles. These should be assigned by the user or by the GCST upon initial review of new issues.

The CONTRIBUTING.md and SUPPORT.md files have also been updated to include links to one another.

See also:
- https://github.com/geoschem/geos-chem/pull/1650
- https://github.com/geoschem/HEMCO/pull/195
- https://github.com/geoschem/GCClassic/pull/32

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question-issue.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+### Name and Institution (Required)
+
+Name:
+Institution:
+
+### Confirm you have reviewed the following documentation
+
+- [ ] [Support guidelines](https://gchp.readthedocs.io/en/latest/reference/SUPPORT.html)https://geos-chem.readthedocs.io/en/stable/reference/SUPPORT.html)
```
Comment (yantosca) [inline .github/ISSUE_TEMPLATE/question-issue.md]: The URL is listed twice here.

**Review by yantosca — CHANGES_REQUESTED** [review]
Looks good to merge after fixing the duplicate URL.

File: .github/ISSUE_TEMPLATE/question-issue.md (hunk 1 above)
Comment (msulprizio) [reply .github/ISSUE_TEMPLATE/question-issue.md]: Now resolved in https://github.com/geoschem/GCHP/pull/293/commits/e19cbea17e0e20ce0a9cc09ab9c0f75f8b317ed7.

Comment (yantosca) [reply .github/ISSUE_TEMPLATE/question-issue.md]: Great, good to merge now!

**Pull request merged**